xdebug_stack.c

```diff
@@ -693,7 +693,7 @@ void xdebug_error_cb(int type, const char *error_filename, const uint error_line
 
 #ifdef PHP_WIN32
 			if (type==E_CORE_ERROR || type==E_CORE_WARNING) {
-				MessageBox(NULL, buffer, error_type_str, MB_OK|ZEND_SERVICE_MB_STYLE);
+				MessageBox(NULL, buffer, error_type_str, MB_OK);
```

Comment:
I've no idea what this did in the first place... I'll do some checking what it was for. I'm curious now.